src/components/overview/LeftPanel.tsx

```diff
@@ -1,59 +1,78 @@
 import CollectionButton from './CollectionButton'
 import { useAutoAnimate } from '@formkit/auto-animate/react'
-import { useCallback, useState } from 'react'
-import data from '@emoji-mart/data'
+import { useCallback } from 'react'
 import useScrollbarStyles from '../../hooks/useScrollbarStyles'
 import useMainBgColour from '../../hooks/useMainBgColour'
 import useBorderColour from '../../hooks/useBorderColour'
-import {error } from "tauri-plugin-log"
+import { error } from 'tauri-plugin-log'
 
 interface Props {
   onOpenSettingsModal: () => void
 }
 
+function SearchBar() {
+  const cancelSearchButtonColour = useColorModeValue('#A0AEC0', '#52525b')
+  const borderColour = useColorModeValue(
+    'primary-light.500',
+    'primary-dark.500'
+  )
+  const { searchValue, setSearchValue } = useApplicationContext()
+
+  return (
+    <Input
+      type="search"
+      maxW={['full']}
+      maxH="32px"
+      variant="brand"
+      placeholder="Search for a macro..."
+      _placeholder={{ opacity: 1, color: borderColour }}
+      sx={{
+        '&::-webkit-search-cancel-button': {
+          WebkitAppearance: 'none',
+          display: 'inline-block',
+          width: '16px',
+          height: '16px',
+          background: `linear-gradient(45deg, rgba(0,0,0,0) 0%,rgba(0,0,0,0) 43%,${cancelSearchButtonColour} 45%,${cancelSearchButtonColour} 55%,rgba(0,0,0,0) 57%,rgba(0,0,0,0) 100%), linear-gradient(135deg, rgba(0,0,0,0) 0%,rgba(0,0,0,0) 43%,${cancelSearchButtonColour} 45%,${cancelSearchButtonColour} 55%,rgba(0,0,0,0) 57%,rgba(0,0,0,0) 100%)`,
+          cursor: 'pointer'
+        }
+      }}
+      value={searchValue}
+      onChange={(event) => setSearchValue(event.target.value)}
+    />
+  )
+}
+
 export default function LeftPanel({ onOpenSettingsModal }: Props) {
   const {
     collections,
     selection,
     onCollectionAdd,
     onCollectionUpdate,
-    changeSelectedCollectionIndex
+    changeSelectedCollectionIndex,
+    isMacroOutputEnabled,
+    changeMacroOutputEnabled,
+    searchValue
   } = useApplicationContext()
   const [parent] = useAutoAnimate<HTMLDivElement>()
   const toast = useToast()
-  const [isMacroOutputEnabled, setIsMacroOutputEnabled] = useState(true)
 
   const onNewCollectionButtonPress = useCallback(() => {
-    const randomCategory =
-      data.categories[
-        Math.floor(Math.random() * (data.categories.length - 3) + 1) // The plus 1 is to avoid selecting the frequent category. The - 3 is to avoid selecting the flags and symbols categories
-      ]
-    let randomEmoji =
-      randomCategory.emojis[
-        Math.floor(Math.random() * randomCategory.emojis.length)
-      ]
-    if (randomEmoji.includes('flag') || randomEmoji.includes('symbols')) {
-      randomEmoji = 'smile'
-    }
-
     onCollectionAdd({
       active: true,
-      icon: `:${randomEmoji}:`,
+      icon: `:😍:`,
       macros: [],
       name: `Collection ${collections.length + 1}`
     })
```

Comment:
Why no random emoji?

Reply:
Consulted with Khang, we looked into it and he concluded the library he used had a really weird typescript setup that didn't work with updated libraries, so he suggested I take it out.

Reply:
Referenced: https://github.com/WootingKb/wooting-macros/issues/206